### PR TITLE
[RHOSINFRA-113] Add ability to build images

### PR DESCRIPTION
**Pull request by fhubik**
Most logic taken from khaleesi so far.

### CODE REVIEW

**Comment by rhosqeauto** [comment]
Can one of the admins verify this patch?

**Comment by yfried-redhat** [comment]
Why is it pushed to stable branch?

File: roles/ospd/overcloud/images/build/tasks/main.yml [hunk 1]
```diff
@@ -1,2 +1,32 @@
 ---
 # TODO: Add images build code here
+- name: build the images on baremetal
+  hosts: undercloud
+  tasks:
+    - name: setup environment vars
+      template: src={{ base_dir }}/khaleesi/playbooks/installer/rdo-manager/templates/build-img-env.j2
```
Comment (yfried-redhat) [inline roles/ospd/overcloud/images/build/tasks/main.yml]: this path won't work

File: roles/ospd/overcloud/images/build/templates/build-img-env.j2 [hunk 2]
```diff
@@ -0,0 +1,41 @@
+export DIB_LOCAL_IMAGE={{ distro.images[distro.name][distro.full_version].guest_image_name }}
+
+{% if product.name == 'rdo' %}
```
Comment (yfried-redhat) [inline roles/ospd/overcloud/images/build/templates/build-img-env.j2]: We are not supporting RDO atm

File: roles/ospd/overcloud/images/build/templates/build-img-env.j2 [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+{% if product.repo_type_override is defined and product.repo_type_override == 'rhsm' %}
+export REG_METHOD=portal
+export REG_USER="{{ distro.rhel.subscription.username }}"
+export REG_PASSWORD="{{ distro.rhel.subscription.password }}"
+export REG_POOL_ID="{{ distro.rhel.subscription.pool }}"
+export REG_REPOS="rhel-7-server-rpms rhel-7-server-extras-rpms rhel-ha-for-rhel-7-server-rpms rhel-7-server-optional-rpms rhel-7-server-openstack-7.0-rpms"
+export NODE_DIST=rhel7
+{%endif %}
+
+{% if product.name == "rdo" and distro.name == "centos" and product.repo.delorean_pin_version is defined and product.repo.delorean is defined %}
```
Comment (yfried-redhat) [inline roles/ospd/overcloud/images/build/templates/build-img-env.j2]: We are not supporting centos atm

**Comment by rhosqeauto** [comment]
Can one of the admins verify this patch?

**Comment by rhosqeauto** [comment]
Can one of the admins verify this patch?

**Comment by rhosqeauto** [comment]
Can one of the admins verify this patch?

**Comment by rhosqeauto** [comment]
Can one of the admins verify this patch?

**Comment by fhubik** [comment]
Please remove this PR as it is proposed for branch stable. I'll submit another one to master with up-to-date code.

**Pull request closed without merging**